mm/Kconfig

```diff
@@ -191,6 +191,49 @@ config MM_KASAN
 		bugs in native code. After turn on this option, Please
 		add -fsanitize=kernel-address to CFLAGS/CXXFLAGS too.
 
+config MM_UBSAN
```

Comment:
let's add MM_UBSAN_TRAP for -fsanitize-undefined-trap-on-error to save the code space